### DISCUSSION TITLE
Install OKD 4.13.11 on Openstack

### DISCUSSION THREAD

**Discussion by balioune** [okd-project/okd#1731] [General]
Hi,
I'm trying to install OKD 4.13 on Openstack using FCOS qcow2 image to deploy the bootstrap server, masters and workers.
After creating the install-config.yaml and generating  ignitions [1] suggest to use userdata to injet ignitions before the first boot.

The main issue is the bootstrap.ign is bigger than the max size that Openstack authorizes.

Please could you suggest a way to run the bootstrap.ign in another way ?
or maybe a way the run bootstrap.ign from another ignition ?

[1] https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-openstack/

**Comment by vrutkovs** [comment]
See https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-virtualbox/#_ignition_config_size_limitations for pointer Ignition solution
